browser/ui/webui/brave_wallet/wallet_panel_ui_browsertest.cc

```diff
@@ -324,6 +324,8 @@ IN_PROC_BROWSER_TEST_F(WalletPanelUIBrowserTest, CustomNetworkInSettings) {
 
   // Chain name for Neon EVM changes to 'Custom Network' in wallet.
   ActivateWalletTab();
+  wallet()->GetController().Reload(content::ReloadType::NORMAL, true);
```

Comment:
should `Reload` be triggered here? It seems like not necessary.

Reply:
Not necessary but it will fix the timing issue. I think the purpose of that test is not for the observer but maybe I'm wrong?

Reply:
The test file already does the reload btw for another test